app/models/manageiq/providers/hawkular/middleware_manager/event_catcher/runner.rb

```diff
@@ -42,13 +42,13 @@ def monitor_events
       event_monitor_running
 
       # Separate alerts from avail updates
-      avail_updates, events = events.partition { |e| !e.kind_of?(::Hawkular::Alerts::Alert) }
+      events, avail_updates = events.partition { |e| !e.kind_of?(::Hawkular::Alerts::Event) }
```

Comment:
Is this file needed for the README.md in any way or was pushed by mistake?

Reply:
Pushed by mistake.